.bazelci/presubmit.yml

```diff
@@ -13,8 +13,8 @@ platforms:
     - "//test/configs:debian8_clang_autoconfig_test"
     - "//rules:debian8-clang-0.3.0-bazel_0.10.0-autoconfig_test"
     - "//container/rbe-debian8:toolchain-test"
-    - "//container/experimental/rbe-debian9:toolchain-test"
-    - "//container/experimental/rbe-ubuntu16_04:toolchain-test"
+    # - "//container/experimental/rbe-debian9:toolchain-test"
```

Comment:
Add a TODO to turn in these tests once issue with CI is resolved

Reply:
Done